xbmc/platform/android/activity/XBMCApp.h

```diff
@@ -276,6 +276,8 @@ class CXBMCApp : public IActivityHandler,
   int64_t m_frameTimeNanos{0};
   float m_refreshRate{0.0f};
 
+  bool isHeadsetPlugged();
```

Comment:
There is no need to expose this method in the header as it does not use any members of CXBMCApp. Please make it an internal helper function.